src/web/styles.scss

```diff
@@ -49,6 +49,19 @@ body {
   padding: 30px 20px;
 }
 
+.table {
+  margin-bottom: 0;
+}
+
+.table th,
+.table td {
+  padding: .5rem;
```

Comment:
You can combine this with the previous `.table` style definition. Use the `&` selector.